.prettierrc

```diff
@@ -5,6 +5,7 @@
   "endOfLine": "lf",
   "htmlWhitespaceSensitivity": "css",
   "insertPragma": false,
+  "jsxBracketSameLine": false,
```

Comment:
Why this change here?

Reply:
This was in Satellite but not in Telesope. I don't think we actually need it.